Source/Test/Operations/UpdateRowOperationTestFixture.cs

```diff
@@ -49,21 +50,21 @@ public void ShouldSetPropertiesForUpdateAllRows()
 		public void ShouldWriteQueryForUpdateSpecificRows()
 		{
 			GivenConditionalRowsToUpdate(SanDiego, Where);
-			ThenQueryValuesAndWhereClauseAre("[Population] = 1234567, [State] = 'Oregon'", "[City] = 'San Diego'");
+			ThenQueryValuesAre(new[] { "[State] = 'Oregon'", "[Population] = 1234567", "[City] = 'San Diego'" });

```

Comment:
Should this also specify a datatype?
```suggestion
			ThenQueryValuesAre(new string[] { "[State] = 'Oregon'", "[Population] = 1234567", "[City] = 'San Diego'" });
```

Reply:
The compiler infers/converts the type for you, so not necessary. I'll leave it as-is.